Stat Modifier feature + ship fixes + scatter quality

LRemplakowski opened the pull request:
1. Fixed
- Not being able to assign crew roles in ship sheet.
- Unnatural characteristic bonus calculation (it was adding 1 instead of unnatural value)

3. Added
- Using psy power on Push strength now allows to choose how many PR you push
- Successful rolls on based on characteristic with Unnatural now get bonus DoS equal to half unnatural value
- Scatter quality script support
- First iteration of modifier system

Modifier system:
Items can now grant Characteristic & Skill bonuses to characters. As long as an item with a modifier is in inventory (so, considered equipped), it will grant its bonus to the character. For now, only Characteristic & Skill modifiers are available, with characteristic allowing for both value & bonus adjustment, and skills being value only.
I plan to add modifiers to specific rolls (like, point-blank range shots, called shots, focus power rolls, etc.) as well as other actor parameters like carrying capacity, movement speed, initiative, etc.

nachofunk commented:
Let me know when you are ready for a review and I will check it out for merging and release.

LRemplakowski commented:
It's ready. Along the way I changed display of weapon ranges in Combat tab to show range brackets for weapons instead of the 'range' value, since range value isn't used for anything other than calculating brackets anyway (for characters at least). Also, added script support for Force weapons.

Next on my personal to do list:
1. Finish the modifier system, to cover as much of character sheet & roll modifiers as possible.
2. Move ammo usage tracking from Item to Character sheet. Mostly to circumvent the issue of ammo usage for tokens with Link Actor Data disabled being shared (so, if you have a generic enemy X, all generic enemy X tokens share the same ammo pool). But I think I can also use that to add extra pen/damage from ammo to be calculated automatically.
